SR2019/2019-04-23-competition-wrap-up.md

```diff
@@ -0,0 +1,52 @@
+---
+to: Student Robotics 2019
+subject: Final competition details
+---
+
+![Competition Photo](https://studentrobotics.org/images/content/news/sr2019/sr2019-photo.jpg)
+
+Hi all,
+
+First off, we'd like to thank you for taking part in Student Robotics 2019 and making it such a successful event! An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/news/2019-04-09-sr2019-tlc-crowned-champion/)
+
+Congratulation again to our winners:
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College Guernsey](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| Rookie Award          | [Eltham College](https://www.elthamcollege.london/)
+| Committee Award       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| Robot and Team Image  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
+| Online Presence       | [Collyer's](https://www.collyers.ac.uk/)
+
+## Event Photos
+
+We've collected together all the media together into 1 album. If you wish to use and republish any of the photos in college material, please be sure to attribute them to [Student Robotics](https://studentrobotics.org).
```

Comment:
Does our media consent actually allow others to use those images? If not, we probably shouldn't encourage it.

I'm also not sure that this attribution is correct, unless we've heard from the individuals concerned that they're willing to hand over the rights to SR (which I wouldn't have expected).

Reply:
> I give myconsent for Student Robotics to use the images and sounds, to store, reproduce, publish and broadcast them in themanner and context and in conjunction with such sounds, images and captions as Student Robotics deems fit.

Hmm, my reading of this is that SR can use the images, it doens't talk about anyone else. Although the final part about "as Student Robotics deems fit" makes me think it _could_ be extended. Thoughts?

Regarding usage rights for photos, hmm I think that's true. The SR website has attributed images to the individuals, so I suspect that's what we'd need to do too. Google Photos lets you see who uploaded a photo, but i've uploaded some on other's behalf, so i'd need to go back and fix that.

Reply:
I see what you mean. Generally I think I'd expect that passing on to a third party for sub-licensing would be explicitly mentioned though.

Reply:
I've updated the wording so that the photos get attributed back to the person who took them. (or at least, who uploaded them to Google). I suspect this is as good as we can get it.

Reply:
It's not the (copyright) attribution I was concerned with here, rather the media consent side. We have permission _for us_ to use the images. I don't think we have permission either for others to use them nor for us to permit others to use them.
As a result, I don't think we should be encouraging the use by others of images which prominently feature people.

Reply:
In hindsight I think I agree, which is really unfortunate. It's also annoying since we setup the photo studio. Thoughts on either emailing each team their own photos, or extracting them all into a separate album, and asking each team to only use the photos of them?

Reply:
I don't think there's an issue with advertising that we have an album of photos that they might wish to look at.

The issue I had was that the wording seemed to be encouraging viewers to use the images themselves, which was what I was worried about. If we're not encouraging further use then it's up to the individuals to check out that they've got the proper rights to an image.

(caveat of course: I am not a Lawyer)

Reply:
I've proposed an alternative wording which just advertises the media without encouraging re-use at https://github.com/srobo/team-emails/pull/20/files#r279157970.

Reply:
https://github.com/srobo/team-emails/pull/20/commits/ab455379ad177ee3c14d214b46ffb1f6212b5826